### ISSUE TITLE
Setting options.uuid in constructor throws Error: Invalid graph! 

### ISSUE THREAD

**Issue by i001962** [amark/gun#1082]
```
const newGun1 = Gun({
  uuid: function () {
    return Math.floor(Math.random() * 4294967296);
  }
})
```
undefined
gun.js:986 Error: Invalid graph! '#0df00f07...' no node.

Yet it looks like the uuid function is set. 

No Error message is thrown if options.uuid are set after the constructor has been initialized.

**Comment by amark** [comment]
@i001962 thanks for reporting this!

Somebody willing to quickly toss a unit test PR in, @rogowski already done all the work of making example showcase templates:  https://github.com/amark/gun/pull/878/files .

**Comment by i001962** [comment]
Yeah I looked at that link and had no idea what you are asking me to do. lol I need a bit more instruction to get that unit test PR thing into next issue I report.

**Comment by i001962** [comment]
Checked back in on this issue today and it is not an issue on the manhattan branch.
Leaving open for the case you want a unit test PR before closing. 

I looked into creating a test but got stuck.
